### PR TITLE
Implement INumber onFraction

### PR DESCRIPTION
**Pull request by Corniel**
By implementing INumber<Fraction> the usage of Fraction can be even further improved. I've chosen to implement most new members explicit.

### CODE REVIEW

File: specs/Qowaiv.Specs/Mathematics/Fraction_INumber_specs.cs [hunk 1]
```diff
@@ -0,0 +1,144 @@
+﻿#if NET8_0_OR_GREATER
+
+using Qowaiv.Specs.TestTools;
+using Qowaiv.TestTools.Numerics;
+
+namespace Mathematics.Fraction_specs;
+
+public class Fraction_as_INumber
```
Comment (renzo-baasdam) [inline specs/Qowaiv.Specs/Mathematics/Fraction_INumber_specs.cs:8]: Inconsistent capitalization of is/Is in function names

File: specs/Qowaiv.Specs/Mathematics/Fraction_INumber_specs.cs [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    [Test]
+    public void Is_always_normal([RandomFraction(Count)] Fraction fraction)
+        => Number.IsNormal(fraction).Should().BeTrue();
+
+    [Test]
+    public void Is_never_subnormal([RandomFraction(Count)] Fraction fraction)
+        => Number.IsSubnormal(fraction).Should().BeFalse();
+
+    [Test]
+    public void maxmaginiute_equal_to_decimal([RandomFraction(3)] Fraction x, [RandomFraction(3)] Fraction y)
```
Comment (renzo-baasdam) [inline specs/Qowaiv.Specs/Mathematics/Fraction_INumber_specs.cs:123]: magnitude, idem below

File: specs/Qowaiv.Specs/_Legacy/Mathematics/FractionOperationsTest.cs [hunk 3]
```diff
@@ -1,210 +0,0 @@
-﻿namespace Qowaiv.UnitTests.Mathematics;
-
-public class FractionOperationsTest
```
Comment (renzo-baasdam) [inline specs/Qowaiv.Specs/_Legacy/Mathematics/FractionOperationsTest.cs:3]: Do the operation tests return somewhere?

Comment (Corniel) [reply specs/Qowaiv.Specs/_Legacy/Mathematics/FractionOperationsTest.cs:3]: Yep. In the tests you reviewed. :)

**Review by renzo-baasdam — APPROVED** [review]

**Pull request merged**